ompi/mca/coll/han/coll_han_dynamic_file.c

```diff
@@ -564,24 +615,24 @@ void mca_coll_han_dump_dynamic_rules(void)
 {
     int nb_entries = 0;
 
-    /* Collective information */
```

Comment:
“information” is an uncountable noun, please remove these changes.

Reply:
thanks for the reminder, sorry